config.rb

```diff
@@ -96,133 +98,9 @@ def deploy_to target
-    /blog/2013/07/rspec-2-14-is-released/
-    /blog/2013/04/rspec-team-changes/
-    /blog/2013/02/rspec-2-13-is-released/
-    /blog/2012/07/mixing-and-matching-parts-of-rspec/
-    /blog/2012/06/constant-stubbing-in-rspec-2-11/
-    /blog/2012/06/rspecs-new-expectation-syntax/
-    /blog/2011/11/recent-rspec-configuration-warnings-and-errors/
-  ].to_set
-
-  def imported_from_myrons_blog?(page)
-    POSTS_IMPORTED_FROM_MYRONS_BLOG.include?(page.url)
-  end
-
-  def disqus_shortname_for(page)
-    if imported_from_myrons_blog?(page)
-      'myronmarston-personal-site'
-    else
-      "rspec"
-    end
-  end
-
-  def disqus_identifier_for(page)
-    path = page.url
-    if imported_from_myrons_blog?(page)
-      path.sub(%r{^/blog/}, "/n/dev-blog/").sub(%r{/$}, '')
-    else
-      path
-    end
-  end
-  alias disqus_path_for disqus_identifier_for
-
-  def disqus_url_for(page)
-    path = disqus_path_for(page)
-    if imported_from_myrons_blog?(page)
-      "http://myronmars.to#{path}"
-    else
-      "http://rspec.info#{path}"
-    end
-  end
+require 'rspec_info/helpers'
+helpers RSpecInfo::Helpers
 
-=begin
-  Borrowed from the EJS gem:
-    https://github.com/sstephenson/ruby-ejs/blob/v1.1.1/lib/ejs.rb#L6-L17
-    https://github.com/sstephenson/ruby-ejs/blob/v1.1.1/lib/ejs.rb#L60-L63
-
-  Copyright (c) 2011 Sam Stephenson
-
-  Permission is hereby granted, free of charge, to any person obtaining
-  a copy of this software and associated documentation files (the
-  "Software"), to deal in the Software without restriction, including
-  without limitation the rights to use, copy, modify, merge, publish,
-  distribute, sublicense, and/or sell copies of the Software, and to
-  permit persons to whom the Software is furnished to do so, subject to
-  the following conditions:
-
-  The above copyright notice and this permission notice shall be
-  included in all copies or substantial portions of the Software.
-
-  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-  MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
-  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
-  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
-  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-=end
-  JS_UNESCAPES = {
-    '\\' => '\\',
-    "'" => "'",
-    'r' => "\r",
-    'n' => "\n",
-    't' => "\t",
-    'u2028' => "\u2028",
-    'u2029' => "\u2029"
-  }
-  JS_ESCAPES = JS_UNESCAPES.invert
-  JS_ESCAPE_PATTERN = Regexp.union(JS_ESCAPES.keys)
-  def js_escape(string)
-    string.gsub(JS_ESCAPE_PATTERN) { |match| '\\' + JS_ESCAPES[match] }
-  end
+RSpecInfo::Helpers.rspec_documentation_latest(source_dir).each do |gem_name, version|
+  proxy "/documentation/latest/#{gem_name}/", "/documentation/#{version}/#{gem_name}/index.html"
```

Comment:
Playing around with this locally, it looks like this works for the root README page for each gem but not for any pages under that. For rspec/rspec-expectations#823, we really just want http://rspec.info/documentation/latest/rspec-expectations/RSpec/Matchers/MatcherProtocol.html to proxy or redirect to http://rspec.info/documentation/3.3/rspec-expectations/RSpec/Matchers/MatcherProtocol.html but that's not working right now.

Can you think of a way to make this work for all the doc pages?

Also, redirects would be better than serving the same content at a second URL.  Of course, with a static site generator you can't choose HTTP status codes to respond with to do a proper redirect but we could use a meta refresh tag.  Thoughts?

/cc @JonRowe